package.json

```diff
@@ -20,22 +21,40 @@
   },
   "homepage": "https://github.com/fec3-galadriel/matt-item-summary#readme",
   "dependencies": {
+    "babel": "^6.23.0",
     "bluebird": "^3.5.5",
     "body-parser": "^1.19.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
The Enzyme adapter should be in `devDependencies`.

Reply:
Weird, airbnb style guide was telling me to move it to dependencies along with enzyme